server/src/security/authorisationService.ts

```diff
@@ -173,14 +175,19 @@ export function getUserRole(userWithType: NonTokenUserWithType): Role | null {
   if (userWithType.type === "ICredential") {
     return OpcoRole
   }
+  const userState = controlUserState(userWithType.value.status)
 
   switch (userWithType.value.type) {
     case "ADMIN":
       return AdminRole
     case "CFA":
       return CfaRole
     case "ENTREPRISE":
-      return RecruiterRole
+      if (userState.error) {
+        return PendingRecruiterRole
+      } else {
+        return RecruiterRole
+      }
```

Comment:
Dans `error` il y a possiblement d'autre choses. C'est la sécurité il faut etre strict:
```suggestion
      if (userState.error) {
        if (userState.reason !== 'VALIDATION') throw Boom.internal('Unexpected'); 
        return PendingRecruiterRole
      } else {
        return RecruiterRole
      }
```

Reply:
En gros l'idée, c'est si ça évolue dans le future comment je force le future dev à venir regarder par ici

Reply:
ok j'ajoute